### DISCUSSION TITLE
Live query regression / behavior change since 3.1.x: delete notification does not contain embedded fields

### DISCUSSION THREAD

**Discussion by denis-itskovich** [orientechnologies/orientdb#9616] [Q&A] [answered]
Is it a known change in live query behavior in 3.1.x compared to 3.0.x, that delete notification does not contain embedded fields?
I described this in issue #9614 and also created a git repository with a junit test for easy reproduction. 

Tried for various 3.0.x versions (3.0.21, 3.0.25, 3.0.37) - for all of them "delete" notifications contain all the data, including embedded fields, but for 3.1.x (tried for 3.1.1, 3.1.6, 3.1.11) - "delete" notifications contains all fields, excepting embedded. Embedded are just `null`.

I tried to find any information on this change in documentation, but could not find any clue. If it's expected, can someone point me to the documentation describing this change and motivation?

Thanks

**Comment by luigidellaquila** [comment, marked as answer]
Hi @denis-itskovich 

it's not a known behaviour, I'd say it's just a bug, I'll check it ASAP

Thanks

Luigi
